02-intermediate-ruby/enumerable.md

```diff
@@ -113,10 +116,10 @@ Use these methods to filter collections by a conditional evaluation. `select` wi
 ```
 
 ## There's so much more to discover
-`Enumerable` is one of the most exciting things about Ruby. It is immensely useful. With it, we can write expressive, highly semantic code that is also concise. Spend some time with the [Enumerable docs](http://ruby-doc.org/core-2.4.0/Enumerable.html). Good knowledge in here will serve you in every application you build, and not just with Ruby. The methods contained in `Enumerable` describe ideas and patterns you'll find in every programming discipline.
+`Enumerable` is one of the most exciting things about Ruby. It is immensely useful. With it, we can write expressive, highly semantic code that is also concise. Spend some time with the [Enumerable docs](http://ruby-doc.org/core-2.4.1/Enumerable.html). Good knowledge in here will serve you in every application you build, and not just with Ruby. The methods contained in `Enumerable` describe ideas and patterns you'll find in every programming discipline.
```

Comment:
If you change this URL to `http://ruby-doc.org/core/Enumerable.html` (leaving out the `-2.4.1`), it will redirect to the most recent version, and we won't have to update this document every cohort.